Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -275,26 +275,34 @@ public enum CommandLineArgumentsResult {
           // Fall through and try to handle the file as an "other file"
         }
 
+        var relative = System.IO.Path.GetFileName(file);
+        bool useRelative = options.UseBaseNameForFileName || relative.StartsWith("-");
```

Comment:
What is the significance of the `-` here?

Reply:
It means it is likely the start of an option. And even if its not, one just gets the relative name of the file, which is a more readable error message